packages/editor/src/client/blocks/embeds/iframe.tsx

```diff
@@ -6,23 +6,43 @@ export interface IframeEmbedProps {
   title?: string
   width?: number
   height?: number
+  styleHeight?: string
+  styleWidth?: string
+  ratio?: boolean
 }
 
-export function IframeEmbed({url, title, width = 300, height = 300}: IframeEmbedProps) {
-  const ratio = width / height
-
+export function IframeEmbed({
+  url,
+  title,
+  width = 300,
+  height = 300,
+  styleHeight,
+  styleWidth,
+  ratio
+}: IframeEmbedProps) {
+  const ratioVal = width / height
+  const styles = {}
+  if (!ratio) {
+    Object.assign(styles, {
+      width: styleWidth,
+      height: styleHeight
+    })
+  }
   return (
     <Box width="100%">
-      <Box position="relative" paddingTop={`${(1 / ratio) * 100}%`} width="100%">
+      <Box
+        position="relative"
+        paddingTop={`${ratio || styleHeight === undefined ? (1 / ratioVal) * 100 + '%' : '0'}`}
+        width="100%"
+        minHeight={'45px'}>
```

Comment:
Why the `minHeight`?

Reply:
because if height if 0 you are not able tp click the edit button if styles are applied

Reply:
is needed in editor

Reply:
we did no spoke about that again for a while, i assume this is ok, plz re-resolve if you dont agree.